src/transformers/configuration_utils.py

```diff
@@ -55,8 +53,8 @@ class PretrainedConfig(object):
 
     def __init__(self, **kwargs):
         # Attributes with defaults
-        self.output_attentions = kwargs.pop("output_attentions", False)
         self.output_hidden_states = kwargs.pop("output_hidden_states", False)
+        self.output_attentions = kwargs.pop("output_attentions", False)
         self.use_cache = kwargs.pop("use_cache", True)  # Not used by all models
```

Comment:
If  we remove the `config.output_attentions` from the documentation, then I think we should also add a deprecation warning so that users are aware this isn't the desired way to control the attentions anymore.